client/js/bar.js

```diff
@@ -59,4 +54,9 @@ function createBarChart(chartValues, chartCategories) {
   }
 }
 
-createBarChart(CHART_VALUES, CHART_CATEGORIES);
+GENRE_ANALYSIS_PROMISE.then((genreAnalysisInfo) => {
+  createBarChart(
+      Object.values(genreAnalysisInfo.genreData),
```

Comment:
My opinion here would be to pass the entire map rather than separating it out. In createBarChart, we start to lose the connection between these two arrays when you're accessing values in one array based on the length of the other array. Open to other thoughts though @geekster777?

Reply:
+1. It makes your function call more concise, and like Jess said, keeps the direct connection between your categories and their values.